docs/cudf/source/cudf_polars/engine_options.md

```diff
@@ -0,0 +1,24 @@
+# GPUEngine Configuration Options
+
+The `polars.GPUEngine` object may be configured in several different ways.
+
+## Parquet Reader Options
+
+Chunked reading is controlled by passing a dictionary of options to the `GPUEngine` object. Details may be found following the links to the underlying `libcudf` reader.
```

Comment:
*nit*: This doesn't tell me why I should use "chunked" reading.

How about:

> Reading large parquet files, especially if they are compressed, can run out of device memory. At the cost of a small drop in performance, the memory used for parquet reads can be reduced by selecting the "chunked" reader. When enabled, parquet files are read in chunks with a limit on the amount of memory that is used.

> To configure the parquet reader, we provide a dictionary of options to the `parquet_options` keyword in the `GPUEngine` object. Valid keys and values are:

...